.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,16 +1,16 @@
-# Pull Request
+# :rocket: Hey, I have created a Pull Request
 
 ## Description of changes
 
 <!-- Fill in a description of what this PR changes/introduces/fixes
      Link to GitHub issues using keywords https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/using-keywords-in-issues-and-pull-requests when necessary
 -->
 
-## Checklist
+## :heavy_check_mark: Checklist
 
 <!-- We follow conventional commit-style PR titles and kebab-case branch names -->
 
-- [ ] I have followed the contribution guidelines for this repository
+- [ ] I have followed the [contribution guidelines](https://github.com/philips-software/amp-devcontainer/blob/main/.github/CONTRIBUTING.md) for this repository
 - [ ] I have added tests for new behavior, and have not broken any existing tests
```

Comment:
Please add a check on having read, understood, and signed-off the Developer Certificate of Origin